client/my-sites/importer/importer-header.jsx

```diff
@@ -71,15 +71,15 @@ export const ImporterHeader = React.createClass( {
 		const { importerState } = this.props.importerStatus;
 
 		if ( includes( startStates, importerState ) ) {
-			return this.translate( 'Start Import', { context: 'verb' } );
+			return this.translate( 'Start', { context: 'verb' } );
```

Comment:
Warning: 30 translations will be lost with this change.
Alternate string suggestion: 
* `translate( 'Start' )`&mdash; translations: **31**. *ES Score: 5.14*

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-stop-during-import)